vhdbuilder/packer/configure-windows-vhd.ps1

```diff
@@ -309,6 +309,11 @@ function Get-SystemDriveDiskInfo {
     }
 }
 
+function Get-DefenderPreferenceInfo {
+    Write-Log "Get preferences for the Windows Defender scans and updates"
+    Write-Log (Get-MpPreference)
```

Comment:
Write-Log (Get-MpPreference)
```
2022-04-28T12:23:29.9980773+08:00: MSFT_MpPreference (ComputerID = "F8734AC6-8C47-C032-C700-2970BF5DB9CB")
```
Use below code
```
Write-Log "Get preferences for the Windows Defender scans and updates"
Get-MpPreference | Write-OutPut
```

Reply:
Or
```
Write-Log(Get-MpPreference | Format-List | Out-String)
```

Reply:
Updated. Thanks for your guidance.